Update community-plugins.json

rouvenjahnke opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin:
https://github.com/rouvenjahnke/study-analytics

## Release Checklist
- [ ] I have tested the plugin on
  - [x]  Windows
  - [ ]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [x] `main.js`
  - [x] `manifest.json`
  - [ ] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
Please remove the `.backup`, `.bak`  and `.original` files from your repo, if you need the file content then you can use the version history to read them.

[.tag {](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/styles.css#L252-L260)
Please don't overwrite core styling in a plugin, add another class/data attribute so it only applies to your plugin.

[text: 'Today\'s Study Time: Loading...'](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L373), [: 'New Session',](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L416), ['Switch to Rec. Sess.' : 'Switch to Recurring Sessions';](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L393), [let toggleText = 'Switch to Stopwatch';](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L389), [notesDiv.createEl('label', { text: 'Session Notes:' });](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L498), [text: 'Add Reflection',](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L507), [text: 'Report Distraction',](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L515), [modal.titleEl.setText('Add Reflection');](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L1640), [statsList.createEl('li', { text: `📝 Modified Files: ${session.modifiedFiles.size}` });](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L1658-L1659), [text: 'Add Reflection',](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L1673), ['Switch to Rec. Sess.' : 'Switch to Recurring Sessions';](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L1719), [this.stopwatchToggle.textContent = 'Switch to Pomodoro';](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L1725), [this.stopwatchToggle.textContent = 'Switch to Stopwatch';](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L1736), [.setName('Pomodoro Duration')](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L1775), [.setName('Short Break Duration')](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L1785), [.setName('Long Break Duration')](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L1795), [.setName('Long Break Interval')](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L1805), [text: 'Report Distraction',](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L515), [.setButtonText('Add Category')](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L1855), [.setName('Notes Folder')](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L1885), [.setName('Focus Path')](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L1895), [.setName('Add Folder-Category Mapping')](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L2128), [containerEl.createEl('h4', { text: 'Tags to Track' });](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L2174), [.setName('Tags to Track')](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L2177), [.setName('Prorate Weekly Goals')](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L2230), [.setName('Prorate Weekly Goals')](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L2230), [name: 'Start New Study Session',](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L2478), [name: 'End Current Study Session',](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L2492), [name: 'Create Daily Study Summary',](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L2498), [name: 'Create Weekly Study Summary',](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L2504), [name: 'Toggle Timer (Start/Pause)',](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L2511), [summaryContent += `## 📊 Daily Statistics\n`;](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L2746-L2758), [summaryContent += `## 📊 Weekly Statistics\n`;](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L3165-L3180) and more:
[Use sentence case in UI](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+sentence+case+in+UI)

[/* Main container styling */](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L738-L1162)
This should be in the `styles.css` file.

[containerEl.createEl('h3', { text: 'Timer Settings' });](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L1772), [containerEl.createEl('h3', { text: 'Categories' });](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L1848), , [containerEl.createEl('h3', { text: 'Tags' });](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L1814), [containerEl.createEl('h3', { text: 'Summary Settings' });](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L1985), [containerEl.createEl('h3', { text: 'Recurring Sessions' });](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L2200), [containerEl.createEl('h4', { text: 'Folder-Category Mappings' });](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L2106), [containerEl.createEl('h4', { text: 'Tags to Track' });](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L2174), [.setName('Prorate Weekly Goals')](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L2230)
For section headings in settings use:
```ts
new Setting(containerEl).setName('name here').setHeading();
```

[containerEl.createEl('h3', { text: 'Timer Settings' });](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L1772)
[Avoid "settings" in settings headings](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Avoid+%22settings%22+in+settings+headings)



[containerEl.createEl('h3', { text: 'Other Settings' });](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L1882)
This section should be at the beginning and not have a header

[new Notice(`Ziel für ${category} erreicht!`);](https://github.com/rouvenjahnke/study-analytics/blob/d1fa672e560bcf73fc5e0bbf350a687be1d81d72/main.ts#L2621)
Please be consistent with the language used, or implement a system for i18n

rouvenjahnke commented:
> Please remove the `.backup`, `.bak` and `.original` files from your repo, if you need the file content then you can use the version history to read them.

@joethei thank you for the response! I made the changes.

in detailed I modified this:
  ✅ Removed backup files - Cleaned up .backup, .bak and .original files from repository
  ✅ Fixed CSS styling - Changed .tag class to .study-flow-tag to avoid overwriting core Obsidian styling
  ✅ Converted UI text to sentence case - Updated all button text, command names, settings labels, and status messages to use proper sentence case (e.g., "Add
  Reflection" → "Add reflection")
  ✅ Moved main container styling to styles.css - Removed the getStyles() method and moved all inline CSS to the dedicated styles.css file
  ✅ Replaced h3/h4 headers with Setting().setHeading() - Updated all settings section headers to use the proper Obsidian API pattern
  ✅ Removed 'Settings' from section headings - Changed "Timer Settings" → "Timer", "Summary Settings" → "Summary", etc.
  ✅ Moved 'Other Settings' to beginning - Reorganized settings layout and removed the "Other Settings" header as requested
  ✅ Made language consistent - Translated German text to English (e.g., "Ziel für ${category} erreicht!" → "Goal for ${category} reached!")

rouvenjahnke commented:
@ Are there any problems with the code of this plugin?

rouvenjahnke commented:
Ok, I'll make the changes in the next weeks.

joethei commented:
[const style = document.createElement('style');](https://github.com/rouvenjahnke/study-analytics/blob/39f38218a0508a6410fa1bec40b93d9be775e076/main.ts#L4051-L4075)
This needs to be in the `styles.css`
